documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -72,6 +72,67 @@ Information and procedures for using a {prodname} Oracle connector are organized
   * xref:debezium-oracle-connector-frequently-asked-questions[]
 endif::product[]
 
+[[oracle-adapter-modes]]
```

Comment:
Should we insert this section at the end of the Overview (which is where it now sits), or as the first topic within the How the connector works assembly? 
I'm slightly partial to the latter option, if only because, in the product version, we could add an entry for it in the preceding list of topics. 
```suggestion
// Type: concept
// ModuleID: descriptions-of-debezium-oracle-connector-adapters
// Title: Descriptions of {prodname} Oracle connector adapters
[[oracle-adapter-modes]]
```

Reply:
What you're saying makes sense. Let's talk about an overhaul -- it's certainly needed. I agree that's out of scope for this PR. Re: tweaking the first text, do you have something in mind?